tests/Tests/Unit/Common/Forms/FormVitalsTest.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+/**
+ * FormVitalsTest.php
+ * @package openemr
+ * @link      http://www.open-emr.org
+ * @author    Stephen Nielson <stephen@nielson.org>
+ * @copyright Copyright (c) 2021 Stephen Nielson <stephen@nielson.org>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+/**
+ * FormVitalsTest.php
+ * @package openemr
+ * @link      http://www.open-emr.org
+ * @author    Stephen Nielson <stephen@nielson.org>
+ * @copyright Copyright (c) 2021 Stephen Nielson <stephen@nielson.org>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
```

Comment:
dbl header :baseball: :)

Reply:
dang, I missed this. losing my code review touch :)

Reply:
Double the fun.. Not sure why phpstorm doesn't flag that as an error for me.